schema.sql

```diff
@@ -64,3 +64,36 @@ ADD COLUMN owners_id INT;
 
 ALTER TABLE animals 
 ADD FOREIGN KEY(owners_id) REFERENCES owners(id) ON DELETE CASCADE;
+
+
+
+CREATE TABLE vets (
+    id BIGSERIAL PRIMARY KEY,
```

Comment:
- [ ] <b>OPTIONAL:</b>   It would be nice to add `NOT NULL` constraint on ID just to make sure that it can't accept null values.

Reply:
thanks